Patch-Source: https://git.deuxfleurs.fr/Deuxfleurs/garage/pulls/556
--
From 6468535a96a853179b4a7211488fb4ae29d8fae5 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 30 Apr 2023 01:01:56 +0200
Subject: [PATCH] Include s3/website test only when k2v is enabled

It depends on k2v feature, so fails to build when k2v is disabled.
---
 src/garage/tests/s3/mod.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/garage/tests/s3/mod.rs b/src/garage/tests/s3/mod.rs
index 623eb66..e47110f 100644
--- a/src/garage/tests/s3/mod.rs
+++ b/src/garage/tests/s3/mod.rs
@@ -3,4 +3,5 @@ mod multipart;
 mod objects;
 mod simple;
 mod streaming_signature;
+#[cfg(feature = "k2v")]
 mod website;
-- 
2.38.4

